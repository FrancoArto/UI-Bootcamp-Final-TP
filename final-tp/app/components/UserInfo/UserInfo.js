'use strict';
import React, { Component } from 'react';
import {
    View,
    Image,
    Text,
    TouchableOpacity,
    Linking
} from 'react-native';
import { styles } from './userInfo.style';
import { Thumbnail, Icon } from 'native-base';


export class UserInfo extends Component {
    constructor(props) {
        super(props);
    }
    render() {
        let userLink = <View></View>;
        let userLocation = <View></View>;
        let formatDescription = this.props.user.description.replace(/\n|\r/g, "").trim(); ///to delete all spacesJump
        let unFormatCreationDate = this.props.user.created_at
        let formatCreationDate =unFormatCreationDate.split(" ", unFormatCreationDate.lenght);
        if(this.props.user.url){
            userLink = <TouchableOpacity onPress={() => Linking.openURL(this.props.user.url)}>
                            <Text style={{color:'blue'}}><Icon type="EvilIcons" name="link"/> {this.props.user.url}</Text>
                         </TouchableOpacity>
        }
        if(this.props.location){
            userLocation = <Text style={styles.statsColor}><Icon type="EvilIcons" name="location" />{this.props.user.location}</Text>
        }
        return (
            <View style={styles.mainContainer}>
                <View style={styles.bannerContainer}>
                    <Image source={{uri: this.props.user.profile_banner_url}}
                           style={styles.banner}
                    />
                </View>
                <View style={styles.userLogoAndNameContainer}>
                    <View>
                        <Thumbnail large source={{uri: this.props.user.profile_image_url}} style={styles.userImage} />
                    </View>
                    <View style={styles.userNameAndHash}>
                        <Text style={styles.userName}>{ this.props.user.name }</Text>
                        <Text style={styles.userDispayedName}>@todonoticias</Text>
                                           
                    </View>
                </View>
                <View style={styles.userDescriptionContainer}>
                    <View style={styles.desciptionView}>
                        <Text style={styles.desciption}>{formatDescription}</Text>
                   </View>
                   <View style={styles.userStats}>
                        {userLocation}
                        {userLink}
                    </View>
                    <View style={styles.userStats}>
                        <Text style={styles.statsColor}><Icon type="EvilIcons" name="calendar" />Se unio el: {formatCreationDate[2]+ ' ' +formatCreationDate[1]+ ' ' +formatCreationDate[5]}</Text>
                    </View>
                    <View style={styles.userStats}>
                        <Text style={styles.statsColor}> <Icon style={styles.iconStyle} type="SimpleLineIcons" name="user-following" /><Text>  {this.props.user.friends_count}  </Text>Siguiendo </Text>
                        <Text style={styles.statsColor}><Icon type="EvilIcons" name="check" /><Text>  {this.props.user.followers_count}  </Text>Seguidores</Text>
                    </View>
                </View>                
            </View>
        )
        
    }
}
