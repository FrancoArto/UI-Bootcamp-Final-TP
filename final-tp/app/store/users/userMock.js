export default {
  "id": 8105922,
  "id_str": "8105922",
  "name": "Clarín",
  "screen_name": "clarincom",
  "location": "Buenos Aires, Argentina",
  "profile_location": null,
  "description": "Noticias de Argentina y el mundo las 24 horas. Información de último momento, opinión, análisis e informes especiales.",
  "url": "http://t.co/1RuczZo0FK",
  "entities": {
    "url": {
      "urls": [
        {
          "url": "http://t.co/1RuczZo0FK",
          "expanded_url": "http://www.clarin.com",
          "display_url": "clarin.com",
          "indices": [
            0,
            22
          ]
        }
      ]
    },
    "description": {
      "urls": []
    }
  },
  "protected": false,
  "followers_count": 2810171,
  "friends_count": 265,
  "listed_count": 10833,
  "created_at": "Fri Aug 10 15:14:11 +0000 2007",
  "favourites_count": 391,
  "utc_offset": null,
  "time_zone": null,
  "geo_enabled": true,
  "verified": true,
  "statuses_count": 301743,
  "lang": "es",
  "status": {
    "created_at": "Thu Dec 20 17:06:00 +0000 2018",
    "id": 1075799489686974500,
    "id_str": "1075799489686974467",
    "text": "Atacaron a balazos el Concejo Municipal de Rosario: \"Con la mafia no se jode\" https://t.co/B8NZ861uoF https://t.co/pliMCVpKd6",
    "truncated": false,
    "entities": {
      "hashtags": [],
      "symbols": [],
      "user_mentions": [],
      "urls": [
        {
          "url": "https://t.co/B8NZ861uoF",
          "expanded_url": "https://www.clarin.com/policiales/atacaron-balazos-concejo-deliberante-rosario-mafia-jode_0_t9aX8sX08.html?utm_term=Autofeed&utm_medium=Social&utm_source=Twitter#Echobox=1545324824",
          "display_url": "clarin.com/policiales/ata…",
          "indices": [
            78,
            101
          ]
        }
      ],
      "media": [
        {
          "id": 1075799486948098000,
          "id_str": "1075799486948098054",
          "indices": [
            102,
            125
          ],
          "media_url": "http://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
          "media_url_https": "https://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
          "url": "https://t.co/pliMCVpKd6",
          "display_url": "pic.twitter.com/pliMCVpKd6",
          "expanded_url": "https://twitter.com/clarincom/status/1075799489686974467/photo/1",
          "type": "photo",
          "sizes": {
            "thumb": {
              "w": 150,
              "h": 150,
              "resize": "crop"
            },
            "large": {
              "w": 1200,
              "h": 630,
              "resize": "fit"
            },
            "small": {
              "w": 680,
              "h": 357,
              "resize": "fit"
            },
            "medium": {
              "w": 1200,
              "h": 630,
              "resize": "fit"
            }
          }
        }
      ]
    },
    "extended_entities": {
      "media": [
        {
          "id": 1075799486948098000,
          "id_str": "1075799486948098054",
          "indices": [
            102,
            125
          ],
          "media_url": "http://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
          "media_url_https": "https://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
          "url": "https://t.co/pliMCVpKd6",
          "display_url": "pic.twitter.com/pliMCVpKd6",
          "expanded_url": "https://twitter.com/clarincom/status/1075799489686974467/photo/1",
          "type": "photo",
          "sizes": {
            "thumb": {
              "w": 150,
              "h": 150,
              "resize": "crop"
            },
            "large": {
              "w": 1200,
              "h": 630,
              "resize": "fit"
            },
            "small": {
              "w": 680,
              "h": 357,
              "resize": "fit"
            },
            "medium": {
              "w": 1200,
              "h": 630,
              "resize": "fit"
            }
          }
        }
      ]
    },
    "source": "<a href=\"https://www.echobox.com\" rel=\"nofollow\">Echobox Social</a>",
    "in_reply_to_status_id": null,
    "in_reply_to_status_id_str": null,
    "in_reply_to_user_id": null,
    "in_reply_to_user_id_str": null,
    "in_reply_to_screen_name": null,
    "geo": null,
    "coordinates": null,
    "place": null,
    "contributors": null,
    "is_quote_status": false,
    "retweet_count": 1,
    "favorite_count": 4,
    "favorited": false,
    "retweeted": false,
    "possibly_sensitive": false,
    "lang": "es"
  },
  "contributors_enabled": false,
  "is_translator": false,
  "is_translation_enabled": true,
  "profile_background_color": "131516",
  "profile_background_image_url": "http://abs.twimg.com/images/themes/theme14/bg.gif",
  "profile_background_image_url_https": "https://abs.twimg.com/images/themes/theme14/bg.gif",
  "profile_background_tile": true,
  "profile_image_url": "http://pbs.twimg.com/profile_images/1053387941752369152/r4pGBF3K_normal.jpg",
  "profile_image_url_https": "https://pbs.twimg.com/profile_images/1053387941752369152/r4pGBF3K_normal.jpg",
  "profile_banner_url": "https://pbs.twimg.com/profile_banners/8105922/1544741225",
  "profile_link_color": "FF0324",
  "profile_sidebar_border_color": "FFFFFF",
  "profile_sidebar_fill_color": "EFEFEF",
  "profile_text_color": "333333",
  "profile_use_background_image": true,
  "has_extended_profile": false,
  "default_profile": false,
  "default_profile_image": false,
  "following": true,
  "follow_request_sent": false,
  "notifications": false,
  "translator_type": "none"
}