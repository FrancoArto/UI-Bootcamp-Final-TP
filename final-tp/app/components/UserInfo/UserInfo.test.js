import React from 'react';

import ShallowRenderer from 'react-test-renderer/shallow';
import { UserInfo } from './UserInfo';
import userMock from '../../store/users/userMock';


it('renders without crashing', () => {
  const user = userMock
  const rendered = new ShallowRenderer()
  rendered.render(
  <UserInfo user={user} />)
  expect(rendered).toMatchSnapshot();
});