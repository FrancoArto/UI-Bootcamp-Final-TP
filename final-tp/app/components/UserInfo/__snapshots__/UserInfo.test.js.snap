// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ReactShallowRenderer {
  "_context": Object {},
  "_element": <UserInfo
    user={
      Object {
        "contributors_enabled": false,
        "created_at": "Fri Aug 10 15:14:11 +0000 2007",
        "default_profile": false,
        "default_profile_image": false,
        "description": "Noticias de Argentina y el mundo las 24 horas. Información de último momento, opinión, análisis e informes especiales.",
        "entities": Object {
          "description": Object {
            "urls": Array [],
          },
          "url": Object {
            "urls": Array [
              Object {
                "display_url": "clarin.com",
                "expanded_url": "http://www.clarin.com",
                "indices": Array [
                  0,
                  22,
                ],
                "url": "http://t.co/1RuczZo0FK",
              },
            ],
          },
        },
        "favourites_count": 391,
        "follow_request_sent": false,
        "followers_count": 2810171,
        "following": true,
        "friends_count": 265,
        "geo_enabled": true,
        "has_extended_profile": false,
        "id": 8105922,
        "id_str": "8105922",
        "is_translation_enabled": true,
        "is_translator": false,
        "lang": "es",
        "listed_count": 10833,
        "location": "Buenos Aires, Argentina",
        "name": "Clarín",
        "notifications": false,
        "profile_background_color": "131516",
        "profile_background_image_url": "http://abs.twimg.com/images/themes/theme14/bg.gif",
        "profile_background_image_url_https": "https://abs.twimg.com/images/themes/theme14/bg.gif",
        "profile_background_tile": true,
        "profile_banner_url": "https://pbs.twimg.com/profile_banners/8105922/1544741225",
        "profile_image_url": "http://pbs.twimg.com/profile_images/1053387941752369152/r4pGBF3K_normal.jpg",
        "profile_image_url_https": "https://pbs.twimg.com/profile_images/1053387941752369152/r4pGBF3K_normal.jpg",
        "profile_link_color": "FF0324",
        "profile_location": null,
        "profile_sidebar_border_color": "FFFFFF",
        "profile_sidebar_fill_color": "EFEFEF",
        "profile_text_color": "333333",
        "profile_use_background_image": true,
        "protected": false,
        "screen_name": "clarincom",
        "status": Object {
          "contributors": null,
          "coordinates": null,
          "created_at": "Thu Dec 20 17:06:00 +0000 2018",
          "entities": Object {
            "hashtags": Array [],
            "media": Array [
              Object {
                "display_url": "pic.twitter.com/pliMCVpKd6",
                "expanded_url": "https://twitter.com/clarincom/status/1075799489686974467/photo/1",
                "id": 1075799486948098000,
                "id_str": "1075799486948098054",
                "indices": Array [
                  102,
                  125,
                ],
                "media_url": "http://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
                "media_url_https": "https://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
                "sizes": Object {
                  "large": Object {
                    "h": 630,
                    "resize": "fit",
                    "w": 1200,
                  },
                  "medium": Object {
                    "h": 630,
                    "resize": "fit",
                    "w": 1200,
                  },
                  "small": Object {
                    "h": 357,
                    "resize": "fit",
                    "w": 680,
                  },
                  "thumb": Object {
                    "h": 150,
                    "resize": "crop",
                    "w": 150,
                  },
                },
                "type": "photo",
                "url": "https://t.co/pliMCVpKd6",
              },
            ],
            "symbols": Array [],
            "urls": Array [
              Object {
                "display_url": "clarin.com/policiales/ata…",
                "expanded_url": "https://www.clarin.com/policiales/atacaron-balazos-concejo-deliberante-rosario-mafia-jode_0_t9aX8sX08.html?utm_term=Autofeed&utm_medium=Social&utm_source=Twitter#Echobox=1545324824",
                "indices": Array [
                  78,
                  101,
                ],
                "url": "https://t.co/B8NZ861uoF",
              },
            ],
            "user_mentions": Array [],
          },
          "extended_entities": Object {
            "media": Array [
              Object {
                "display_url": "pic.twitter.com/pliMCVpKd6",
                "expanded_url": "https://twitter.com/clarincom/status/1075799489686974467/photo/1",
                "id": 1075799486948098000,
                "id_str": "1075799486948098054",
                "indices": Array [
                  102,
                  125,
                ],
                "media_url": "http://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
                "media_url_https": "https://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
                "sizes": Object {
                  "large": Object {
                    "h": 630,
                    "resize": "fit",
                    "w": 1200,
                  },
                  "medium": Object {
                    "h": 630,
                    "resize": "fit",
                    "w": 1200,
                  },
                  "small": Object {
                    "h": 357,
                    "resize": "fit",
                    "w": 680,
                  },
                  "thumb": Object {
                    "h": 150,
                    "resize": "crop",
                    "w": 150,
                  },
                },
                "type": "photo",
                "url": "https://t.co/pliMCVpKd6",
              },
            ],
          },
          "favorite_count": 4,
          "favorited": false,
          "geo": null,
          "id": 1075799489686974500,
          "id_str": "1075799489686974467",
          "in_reply_to_screen_name": null,
          "in_reply_to_status_id": null,
          "in_reply_to_status_id_str": null,
          "in_reply_to_user_id": null,
          "in_reply_to_user_id_str": null,
          "is_quote_status": false,
          "lang": "es",
          "place": null,
          "possibly_sensitive": false,
          "retweet_count": 1,
          "retweeted": false,
          "source": "<a href=\\"https://www.echobox.com\\" rel=\\"nofollow\\">Echobox Social</a>",
          "text": "Atacaron a balazos el Concejo Municipal de Rosario: \\"Con la mafia no se jode\\" https://t.co/B8NZ861uoF https://t.co/pliMCVpKd6",
          "truncated": false,
        },
        "statuses_count": 301743,
        "time_zone": null,
        "translator_type": "none",
        "url": "http://t.co/1RuczZo0FK",
        "utc_offset": null,
        "verified": true,
      }
    }
  />,
  "_forcedUpdate": false,
  "_instance": UserInfo {
    "context": Object {},
    "props": Object {
      "user": Object {
        "contributors_enabled": false,
        "created_at": "Fri Aug 10 15:14:11 +0000 2007",
        "default_profile": false,
        "default_profile_image": false,
        "description": "Noticias de Argentina y el mundo las 24 horas. Información de último momento, opinión, análisis e informes especiales.",
        "entities": Object {
          "description": Object {
            "urls": Array [],
          },
          "url": Object {
            "urls": Array [
              Object {
                "display_url": "clarin.com",
                "expanded_url": "http://www.clarin.com",
                "indices": Array [
                  0,
                  22,
                ],
                "url": "http://t.co/1RuczZo0FK",
              },
            ],
          },
        },
        "favourites_count": 391,
        "follow_request_sent": false,
        "followers_count": 2810171,
        "following": true,
        "friends_count": 265,
        "geo_enabled": true,
        "has_extended_profile": false,
        "id": 8105922,
        "id_str": "8105922",
        "is_translation_enabled": true,
        "is_translator": false,
        "lang": "es",
        "listed_count": 10833,
        "location": "Buenos Aires, Argentina",
        "name": "Clarín",
        "notifications": false,
        "profile_background_color": "131516",
        "profile_background_image_url": "http://abs.twimg.com/images/themes/theme14/bg.gif",
        "profile_background_image_url_https": "https://abs.twimg.com/images/themes/theme14/bg.gif",
        "profile_background_tile": true,
        "profile_banner_url": "https://pbs.twimg.com/profile_banners/8105922/1544741225",
        "profile_image_url": "http://pbs.twimg.com/profile_images/1053387941752369152/r4pGBF3K_normal.jpg",
        "profile_image_url_https": "https://pbs.twimg.com/profile_images/1053387941752369152/r4pGBF3K_normal.jpg",
        "profile_link_color": "FF0324",
        "profile_location": null,
        "profile_sidebar_border_color": "FFFFFF",
        "profile_sidebar_fill_color": "EFEFEF",
        "profile_text_color": "333333",
        "profile_use_background_image": true,
        "protected": false,
        "screen_name": "clarincom",
        "status": Object {
          "contributors": null,
          "coordinates": null,
          "created_at": "Thu Dec 20 17:06:00 +0000 2018",
          "entities": Object {
            "hashtags": Array [],
            "media": Array [
              Object {
                "display_url": "pic.twitter.com/pliMCVpKd6",
                "expanded_url": "https://twitter.com/clarincom/status/1075799489686974467/photo/1",
                "id": 1075799486948098000,
                "id_str": "1075799486948098054",
                "indices": Array [
                  102,
                  125,
                ],
                "media_url": "http://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
                "media_url_https": "https://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
                "sizes": Object {
                  "large": Object {
                    "h": 630,
                    "resize": "fit",
                    "w": 1200,
                  },
                  "medium": Object {
                    "h": 630,
                    "resize": "fit",
                    "w": 1200,
                  },
                  "small": Object {
                    "h": 357,
                    "resize": "fit",
                    "w": 680,
                  },
                  "thumb": Object {
                    "h": 150,
                    "resize": "crop",
                    "w": 150,
                  },
                },
                "type": "photo",
                "url": "https://t.co/pliMCVpKd6",
              },
            ],
            "symbols": Array [],
            "urls": Array [
              Object {
                "display_url": "clarin.com/policiales/ata…",
                "expanded_url": "https://www.clarin.com/policiales/atacaron-balazos-concejo-deliberante-rosario-mafia-jode_0_t9aX8sX08.html?utm_term=Autofeed&utm_medium=Social&utm_source=Twitter#Echobox=1545324824",
                "indices": Array [
                  78,
                  101,
                ],
                "url": "https://t.co/B8NZ861uoF",
              },
            ],
            "user_mentions": Array [],
          },
          "extended_entities": Object {
            "media": Array [
              Object {
                "display_url": "pic.twitter.com/pliMCVpKd6",
                "expanded_url": "https://twitter.com/clarincom/status/1075799489686974467/photo/1",
                "id": 1075799486948098000,
                "id_str": "1075799486948098054",
                "indices": Array [
                  102,
                  125,
                ],
                "media_url": "http://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
                "media_url_https": "https://pbs.twimg.com/media/Du4B7v0WwAYZ4qc.jpg",
                "sizes": Object {
                  "large": Object {
                    "h": 630,
                    "resize": "fit",
                    "w": 1200,
                  },
                  "medium": Object {
                    "h": 630,
                    "resize": "fit",
                    "w": 1200,
                  },
                  "small": Object {
                    "h": 357,
                    "resize": "fit",
                    "w": 680,
                  },
                  "thumb": Object {
                    "h": 150,
                    "resize": "crop",
                    "w": 150,
                  },
                },
                "type": "photo",
                "url": "https://t.co/pliMCVpKd6",
              },
            ],
          },
          "favorite_count": 4,
          "favorited": false,
          "geo": null,
          "id": 1075799489686974500,
          "id_str": "1075799489686974467",
          "in_reply_to_screen_name": null,
          "in_reply_to_status_id": null,
          "in_reply_to_status_id_str": null,
          "in_reply_to_user_id": null,
          "in_reply_to_user_id_str": null,
          "is_quote_status": false,
          "lang": "es",
          "place": null,
          "possibly_sensitive": false,
          "retweet_count": 1,
          "retweeted": false,
          "source": "<a href=\\"https://www.echobox.com\\" rel=\\"nofollow\\">Echobox Social</a>",
          "text": "Atacaron a balazos el Concejo Municipal de Rosario: \\"Con la mafia no se jode\\" https://t.co/B8NZ861uoF https://t.co/pliMCVpKd6",
          "truncated": false,
        },
        "statuses_count": 301743,
        "time_zone": null,
        "translator_type": "none",
        "url": "http://t.co/1RuczZo0FK",
        "utc_offset": null,
        "verified": true,
      },
    },
    "refs": Object {},
    "state": null,
    "updater": Updater {
      "_callbacks": Array [],
      "_renderer": [Circular],
    },
  },
  "_newState": null,
  "_rendered": <Component
    style={
      Object {
        "backgroundColor": "white",
        "borderBottomColor": "#AAA",
        "borderBottomWidth": 0.5,
        "flex": 1,
      }
    }
  >
    <Component
      style={
        Object {
          "flex": 22,
        }
      }
    >
      <Image
        source={
          Object {
            "uri": "https://pbs.twimg.com/profile_banners/8105922/1544741225",
          }
        }
        style={
          Object {
            "height": 296.44444444444446,
            "width": 750,
          }
        }
      />
    </Component>
    <Component
      style={
        Object {
          "flex": 20,
          "flexDirection": "row",
          "marginLeft": 11.538461538461538,
        }
      }
    >
      <Component>
        <Styled(Thumbnail)
          large={true}
          source={
            Object {
              "uri": "http://pbs.twimg.com/profile_images/1053387941752369152/r4pGBF3K_normal.jpg",
            }
          }
          style={
            Object {
              "borderColor": "white",
              "borderWidth": 3,
            }
          }
          virtual={undefined}
        />
      </Component>
      <Component
        style={
          Object {
            "marginLeft": 34.09090909090909,
            "marginTop": 75,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "fontSize": 41.666666666666664,
              "fontWeight": "bold",
            }
          }
        >
          Clarín
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#9c9c9c",
            }
          }
        >
          @todonoticias
        </Text>
      </Component>
    </Component>
    <Component
      style={
        Object {
          "flex": 35,
          "marginLeft": 25,
        }
      }
    >
      <Component
        style={
          Object {
            "marginTop": 13.636363636363637,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#5D5D5D",
              "fontSize": 28.846153846153847,
              "fontWeight": "400",
            }
          }
        >
          Noticias de Argentina y el mundo las 24 horas. Información de último momento, opinión, análisis e informes especiales.
        </Text>
      </Component>
      <Component
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 5,
          }
        }
      >
        <Component />
        <TouchableOpacity
          activeOpacity={0.2}
          onPress={[Function]}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "blue",
              }
            }
          >
            <Styled(Icon)
              name="link"
              type="EvilIcons"
              virtual={undefined}
            />
             
            http://t.co/1RuczZo0FK
          </Text>
        </TouchableOpacity>
      </Component>
      <Component
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 5,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#989898",
              "fontSize": 26.785714285714285,
            }
          }
        >
          <Styled(Icon)
            name="calendar"
            type="EvilIcons"
            virtual={undefined}
          />
          Se unio el: 
          10 Aug 2007
        </Text>
      </Component>
      <Component
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 5,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#989898",
              "fontSize": 26.785714285714285,
            }
          }
        >
           
          <Styled(Icon)
            name="user-following"
            style={
              Object {
                "fontSize": 20,
              }
            }
            type="SimpleLineIcons"
            virtual={undefined}
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
          >
              
            265
              
          </Text>
          Siguiendo 
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#989898",
              "fontSize": 26.785714285714285,
            }
          }
        >
          <Styled(Icon)
            name="check"
            type="EvilIcons"
            virtual={undefined}
          />
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
          >
              
            2810171
              
          </Text>
          Seguidores
        </Text>
      </Component>
    </Component>
  </Component>,
  "_rendering": false,
  "_updater": Updater {
    "_callbacks": Array [],
    "_renderer": [Circular],
  },
}
`;
